### ISSUE TITLE
`@Routable` built in handlers (post) does not handle Mongo Duplicate Error

### ISSUE THREAD

**Issue by etsuo** [sakuraapi/api#82]
```
{ MongoError: E11000 duplicate key error collection: user.users index: email_1_domain_1 dup key: { : "redacted@redacted.com", : "default" }
```

**Comment by etsuo** [comment]
fixed in 0.7.0-4

**Comment by etsuo** [comment]
weird... the commit didn't reference here... just in case it doesn't catch up: 1dc4fbca79accf498bbca2dacc93485fc1daf57a
